### PR TITLE
[#209] Preferred browser setting view 

### PR DESCRIPTION
**Pull request by BrianLitwin**
Connected to #209 
Continuing the refactor of the button bindings, very similar to #762, #772, and #779, just one button to refactor this time.

### CODE REVIEW

**Comment by devinreams** [comment]
Thanks for your continued contributions, @BrianLitwin 😁

**Comment by devinreams** [comment]
Two UI specs are "failing" (but the functionality is working as expected) based on some assert/expectations from the test suite about buttons. I took a quick glance but wasn't clever enough to see how to fix this myself.

In case its not obvious to @BrianLitwin or others, @isabelrios may have a speedy idea/fix we can drop in here to get everything green again. ✅

**Comment by BrianLitwin** [comment]
I'll look at this!

File: lockbox-ios/View/PreferredBrowserSettingView.swift [hunk 1]
```diff
@@ -95,6 +95,14 @@ extension PreferredBrowserSettingView: PreferredBrowserSettingViewProtocol {
                 .disposed(by: self.disposeBag)
         }
     }
+    
+    var onSettingsButtonPressed: ControlEvent<Void>? {
+        if let button = self.navigationItem.leftBarButtonItem?.customView as? UIButton {
+            return button.rx.tap
+        }
+
```
Comment (joeyg) [inline lockbox-ios/View/PreferredBrowserSettingView.swift]: Nit: Watch out for leaving an extra tab on these empty lines

**Comment by BrianLitwin** [comment]
I've got some ideas about getting these tests green, but I'm going to close this for now until I've got more time to look at it and get them passing again.

**Pull request closed without merging**